install/k8s/k8s1.6/contiv.yaml

```diff
@@ -142,6 +144,8 @@ spec:
         # Runs netplugin container on each Kubernetes node.  This
         # container programs network policy and routes on each
         # host.
+        - name: sinatra-exporter
+          image: arshiagg/sinatra
         - name: contiv-netplugin
```

Comment:
1. upload image under contiv and use version, ```__CONTIV_VERSION__ ```
2. @neelimamukiri can we make this configurable ?